### ISSUE TITLE
Reference to missing file in combination with --disable-local-file-access reports blocked access, even if file would be allowed if present

### ISSUE THREAD

**Issue by sandersaares** [wkhtmltopdf/wkhtmltopdf#3846]
Consider a broken reference to a file C:\Content\x.png present in the source HTML.

Normally when the app attempts to load this reference, you get the "missing media" behavior:

```
Warning: Failed to load file:///C:/Content/x.png (ignore)
```

However, if you combine it with `--disable-local-file-access` and `--allow C:\Content` you get:

```
Warning: Blocked access to file
```

There are two problems with this:

1. The real issue is that the reference is broken - it would be allowed if the file were present!
1. There is no filename printed at the end of this message, so it is not possible to know which reference is broken even if you know that this error can occur due to broken reference.

For comparison, if an existing file is blocked, it is named:

```
Warning: Blocked access to file C:/Content/y.png
```

Used v0.12.4 on Windows.

**Comment by PhilterPaper** [comment]
Have you tried `--allow C:/Content` and `--allow C:\\Content` to confirm that it's not getting indigestion from that single \ escape character? You never know with mixing C-style coding with Windows.

**Comment by sandersaares** [comment]
Yes, I experimented with different path styles and concluded that they did not appear relevant to the issue at hand.

**Comment by Asepdadan** [comment]
how to open blocked access file wkhtmltopdf?
--disable-local-file-access not work

**Comment by Asepdadan** [comment]
> Have you tried `--allow C:/Content` and `--allow C:\\Content` to confirm that it's not getting indigestion from that single \ escape character? You never know with mixing C-style coding with Windows.

can you show me complete command in windows?

**Comment by PhilterPaper** [comment]
I don't know what the OP used, or what's in the HTML file. According to the documentation, `--disable-local-file-access` should prohibit all attempts at accessing local PC files (i.e., not via a URL), _except_ those directories whitelisted on the command line with the `--allow` flag. This makes sense, as you don't want a random HTML file rooting around in your PC's files!

My point above was that giving a directory name on the command line in the form such as `C:\Content` (for `--allow`) might give the program indigestion, as `"\"` is normally an **escape** indicator. As the `--allow` flag and its value are just passed to the C++ code, it _should_ be safe to say `--allow C:/Content`. Possibly `--allow C:\\Content` will work on the command line, too.

Thank you very much, Bill Gates. You just _had_ to make DOS 2.0 directory separators different from Unix, didn't you? The good news is that most programming languages accept either `\` or `/` for directory separators. You just have to be careful when using `\` that it doesn't get treated as an escape character.
